doc/man3/OCSP_resp_find_status.pod

```diff
@@ -100,6 +104,23 @@ OCSP_single_get0_status(). If B<sec> is non-zero it indicates how many seconds
 leeway should be allowed in the check. If B<maxsec> is positive it indicates
 the maximum age of B<thisupd> in seconds.
 
+OCSP_basic_verify() checks that the basic response message B<bs> is correctly
+signed (unless the B<OCSP_NOSIGS> flag is set) and that the signer certificate
+can be validated (unless the <OCSP_NOVERIFY> flag is set). It takes B<st> as the
+trusted store and B<certs> as a set of untrusted intermediate certificates.
+The function tries to find the signer certificate of the response in B<cert>.
+If not found and the B<OCSP_NOINTERN> flag is not set, it also tries any
+certificates the responder may have included in B<bs>.
+It then validates the signer certificate unless it was found in B<certs>
+and the B<OCSP_TRUSTOTHER> flag is set. Otherwise both sources of untrusted
+certificates are used for constructing the validation path for the signer
+certificate unless the B<OCSP_NOCHAIN> flag is set. After successful
+path validation the function immediately yields a positive result if the
+B<OCSP_NOCHECKS> flag is set. Else it verifies that the signer certificate
+meets the OCSP issuer criteria using B<ocsp_check_issuer()>. If the latter
```

Comment:
`ocsp_check_issuer` is an internal static function, so it probably shouldn't appear in the public API docs.

Reply:
Good point - I removed the reference to the internal function.